src/main/java/org/interledger/setup/model/Receiver.java

```diff
@@ -0,0 +1,18 @@
+package org.interledger.setup.model;
+
+import javax.money.CurrencyUnit;
+
+import org.interledger.ilp.InterledgerAddress;
+
+public interface Receiver {
+  
+  InterledgerAddress getAccount();
+  
+  CurrencyUnit getCurrencyUnit();
+
+  int getPrecision();

```

Comment:
Hmmm, good point. Need to think about this carefully because it could be that we need a custom factory for CurrencyUnit.

It currently uses the default and fetches the CurrencyUnit based on currency code. This means it's possible that the system values for precision and scale are different to those supported by the ledger.

I suggest we leave that in for now and I'll work on a complete solution, possibly involving a custom CurrencyUnit factory.